pkg/version/version.go

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package version
+
+var (
+	// COMMIT contains the hash of the VCS commit
+	COMMIT string
+	// SHORTCOMMIT contains the shorten version of the VCS commit's hash
+	SHORTCOMMIT string
```

Comment:
These variables need changes in Makefile to fill them up: [example](https://github.com/openshift/external-dns-operator/blob/34fff71eaac02401ad42dffe99f62397e3f4aec7/Makefile#L70-L72).